{"name": "test_login_fail[data2]", "status": "passed", "parameters": [{"name": "data", "value": "{'id': 3, 'title': '6位错误验证码', 'url': 'http://192.168.3.157:8069/dongwang-app/app/user/login', 'json': {'type': '1', 'flashToken': '', 'deviceId': 'aaa', 'device': '1', 'smsCode': '123456', 'phone': '113157121093'}, 'expected': {'status_code': 500, 'message': '验证码输入错误,请重新输入'}}"}], "start": 1614072749897, "stop": 1614072749930, "uuid": "af71aea1-34b5-4626-8b5b-b20ce7029343", "historyId": "84c7f37425acf02e7d6820b030f03f6a", "testCaseId": "62fb4c4c1bc44f1f23ceb53be330b5b0", "fullName": "case.test_login.TestLogin#test_login_fail", "labels": [{"name": "story", "value": "错误验证码"}, {"name": "feature", "value": "登录模块"}, {"name": "parentSuite", "value": "case"}, {"name": "suite", "value": "test_login"}, {"name": "subSuite", "value": "TestLogin"}, {"name": "host", "value": "SC-201907251023"}, {"name": "thread", "value": "16284-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "case.test_login"}]}